### DISCUSSION TITLE
[Projects beta] Drag and drop order while sorting is active

### DISCUSSION THREAD

**Discussion by peterlauri** [community/community#11860] [Projects and Issues]
When you have no sorting activated you can drag and drop items in the list up and down. As soon as you turn on a sorting filter, that possibility goes away. My use case is that we have a set of High priority issues, but we want be able to sort them a bit more granular.

https://github.com/github/feedback/discussions/8895 would partially solve our issue, as we can sort by "Priority" and then by "Status".

**Comment by shmup** [comment]
It's nice that you made this discussion, heh. Helped me figure it out.

The UX isn't obvious, because on my new project, it's sorted by default. I think a nicer UX would be, instead of having to "opt in" to the unsorted option, merely trying to drag and drop would change it to unsorted.
